### ISSUE TITLE
Question about normalization

### ISSUE THREAD

**Issue by zqh0253** [autonomousvision/stylegan_xl#41]
Hi, 
I notice that the normalization of input images is different from traditional normalization way ( x = (x-mean)/std. ):
```
def norm_with_stats(x, stats):
    x_ch0 = torch.unsqueeze(x[:, 0], 1) * (0.5 / stats['mean'][0]) + (0.5 - stats['std'][0]) / stats['mean'][0]
    x_ch1 = torch.unsqueeze(x[:, 1], 1) * (0.5 / stats['mean'][1]) + (0.5 - stats['std'][1]) / stats['mean'][1]
    x_ch2 = torch.unsqueeze(x[:, 2], 1) * (0.5 / stats['mean'][2]) + (0.5 - stats['std'][2]) / stats['mean'][2]
    x = torch.cat((x_ch0, x_ch1, x_ch2), 1)
    return x
```

Is there any paper or previous work introduces this kind of normalization method?

**Comment by woctezuma** [comment]
It is a bit weird as the normalization amounts to:
```
x_offset = (x+1)/2
x_norm = (x_offset - std) / mean
```

**Comment by xl-sr** [comment]
it's unnormalization + normalization in a single op. But currently it is somewhat cryptic, I made a push fixing it :)

**Comment by woctezuma** [comment]
I see.

- 9a6de7ed70e7980c04298da3916a3f9492322fba

So "un-normalization" is:

https://github.com/autonomousvision/stylegan_xl/blob/9a6de7ed70e7980c04298da3916a3f9492322fba/pg_modules/projector.py#L115

And "normalization" is:

https://github.com/autonomousvision/stylegan_xl/blob/9a6de7ed70e7980c04298da3916a3f9492322fba/pg_modules/projector.py#L105

The small bug was that mean and std were exchanged, right?

**Comment by xl-sr** [comment]
correct :)
